setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name='kubeflow-fairing',
-    version='0.7.0.1',
+    version='0.7.1',
```

Comment:
Would it make sense to separate out the "version up" from this PR?

Reply:
@abhi-g Thanks! Updated.